tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -251,6 +251,26 @@ def mean_autocorrelation(x):
         return np.nanmean(r / var)
 
 
+def memoize(f):
+    cache = {}
```

Comment:
I dont like that the cache is not size limited. have you looked at some caching libraries?

https://pythonhosted.org/cachetools/